_content/tour/ukr/maps.article

```diff
@@ -1,24 +1,24 @@
-Maps
-A map is a data structure that provides support for storing and accessing data based on a key.
+Мапи (Maps)
```

Comment:
Is it intentional that in the "Translate slices article," maps were translated? Perhaps you forgot to revert it because maps were translated in PR #13.

Reply:
Yeah, sorry I just cherry-picked and pushed this branch too. Since we have comments already I will just keep this commit here.

Reply:
![image](https://github.com/GolangUA/ardanlabs-gotour/assets/32271530/6cedcbfd-7c12-4c67-889b-20777ed50143)

Reply:
😅